yo/app/scripts/directives/transformation.js

```diff
@@ -1132,25 +1132,44 @@ angular.module('dmpApp')
         }
 
         /**
-         * Send a single transformation to the server
+         * This actually sends a Transformation
+         * @param {boolean} persist - Should the data be persisted or just previewed
          */
-        $scope.sendTransformation = function() {
+        function sendTransformationsInternal(persist) {
```

Comment:
I know, 'single transformation preview' is disable since a while. However, the function itself isn't really "interested", whether it gets a single mapping (with a transformation) or multiple mappings. It just creates request and executes it. So this was just a clean-up of duplicated code (since the bodies of both methods were very equal before) ;)

So, I'm not sure, whether I need to mention that one method was called with one mapping before and the other one with (possibly) multiple mappings ...